.github/workflows/build_and_test.yml

```diff
@@ -182,3 +182,85 @@ jobs:
                   MKL_NUM_THREADS: 1
                   NUMEXPR_NUM_THREADS: 1
                   MPLBACKEND: "Agg"
+
+    test-dev:
+        name: Test with dev threeML/astromodels
+        needs: skip_duplicate
+        if: ${{ needs.skip_duplicate.outputs.should_skip == 'false' }}
+        strategy:
+            matrix:
+                os: ["ubuntu-latest", "macos-latest"]
+                python-version: [3.7]
+            fail-fast: false
+        runs-on: ${{ matrix.os }}
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: XCode
+              uses: maxim-lobanov/setup-xcode@v1
+              with:
+                  xcode-version: '12.4'
+              if: runner.os == 'macOS'
+            - name: Cache conda
+              uses: actions/cache@v2
+              with:
+                  path: ~/conda_pkgs_dir
+                  key: conda-${{ matrix.os }}-python-${{ matrix.python-version }}-${{ hashFiles('environment-ci-new.yml') }}
+            - name: Setup Miniconda
+              uses: conda-incubator/setup-miniconda@v2
+              with:
+                  auto-update-conda: true
+                  auto-activate-base: false
+                  activate-environment: test_env
+                  python-version: ${{ matrix.python-version }}
+                  channels: conda-forge, xspecmodels, threeml, defaults
+                  environment-file: ci/environment.yml
+                  mamba-version: "*"
+                  
+            - name: Init Env
+              shell: bash -l {0}
+              run: |
+                  # Make sure we fail in case of error
+                  if [[ ${{matrix.os}} == ubuntu-latest ]];
+                  then
+                  miniconda_os=Linux
+                  compilers="gcc_linux-64 gxx_linux-64"
+                  else  # osx
+                  miniconda_os=MacOSX
+                  compilers="clang_osx-64 clangxx_osx-64 gfortran_osx-64"
+                  fi
+
+                  echo "HOME= ${HOME}"
+                  echo "Python version: ${{matrix.python-version}}"
+
+                  mamba install pytest codecov pytest-cov git astromodels threeML numba reproject root flake8
+                  pip install --no-binary :all: root_numpy
```

Comment:
Should we change line 237 with `mamba install -c conda-forge root_numpy` as well? Since `pip install --no-binary :all: root_numpy` won't succeed while installation.

Reply:
Sure, can you do it?

Reply:
Sure, done!